test/box-luatest/upgrade_to_3_0_0_test.lua

```diff
@@ -20,9 +16,29 @@ end)
 
 g.test_new_replicaset_uuid_key = function(cg)
     cg.server:exec(function()
+        local mkversion = require('internal.mkversion')
+        t.assert_equals(box.space._schema:get{'version'}, {'version', 2, 11, 0})
+
+        -- Test, that schema is broadcasted properly.
+        local version
+        local version_no = 0
+        box.watch('box.status', function(_, status)
+            t.assert_not_equals(status.schema_version, nil)
+            version = mkversion.from_string(status.schema_version)
+            version_no = version_no + 1
+        end)
+
+        box.cfg{read_only = true}
+        t.helpers.retrying({}, function() t.assert_ge(version_no, 1) end)
+        t.assert_equals(version, mkversion.new(2, 11, 0))
+
+        box.cfg{read_only = false}
+        box.schema.upgrade()
+
         local _schema = box.space._schema
         t.assert_equals(_schema:get{'cluster'}, nil)
         t.assert_equals(_schema:get{'replicaset_uuid'}.value,
                         box.info.replicaset.uuid)
+        t.assert_equals(version, mkversion.get_latest())
```

Comment:
Nit: It is mixed with another test case (regarding names). I think it would be more clear if we split them.

Reply:
Sure, I separated it from schema checking test.